sys-boot/refind/refind-0.11.4.ebuild

```diff
@@ -0,0 +1,187 @@
+		ewarn
+	fi
+}
+
+pkg_setup() {
+	if use x86; then
+		export EFIARCH=ia32
+		export BUILDARCH=ia32
+	elif use amd64; then
+		export EFIARCH=x64
+		export BUILDARCH=x86_64
+	fi
+}
+
+src_prepare() {
+	default
+
+	# bug 598647 - PIE not supported
+	sed -e '/^CFLAGS/s:$: -fno-PIE:' -i Make.common || die
+
+	# Update Makefile
+	sed -e "s:^\(\s*\)\. \./edksetup\.sh[^&]*:\1build -a \$(UC_ARCH) -p MdePkg/MdePkg.dsc :" \
+		-i Makefile || die "Could not update Makefile"
+
+	# Prepare UDK workspace
+	if ! use gnuefi; then
+		mkdir "${UDK_WORKSPACE}" || die
+		ln -s "${EPREFIX}"/usr/share/udk/{Mde,IntelFramework}{,Module}Pkg \
+			"${UDK_WORKSPACE}" || die "Could not link UDK files"
+			
+		# HFS filesystem is not working with UDK
+		# See https://sourceforge.net/p/refind/discussion/general/thread/4e203f0a10/
+		sed -e "/RefindPkg\/filesystems\/hfs\.inf/d" -i RefindPkg.dsc || die
+		if use hfs; then
+			eerror "hfs use flag ignored because not working with UDK"
+		fi
+	fi
+}
+
+src_configure() {
+	if ! use gnuefi; then
+		. udk-workspace "${UDK_WORKSPACE}" || die
+		sed -e "s:^\(MAX_CONCURRENT_THREAD_NUMBER\s*=\).*$:\1 $(makeopts_jobs):" \
+			-i "${UDK_WORKSPACE}/Conf/target.txt" || die "Failed to configure target file"
+		sed -e "s:\(_\(CC\|ASM\|PP\|VFRPP\|ASLCC\|ASLPP\|DLINK\)_PATH\s*=\).*$:\1 $(tc-getCC):" \
+			-e "s:\(_ASLDLINK_PATH\s*=\).*$:\1 $(tc-getLD):" \
+			-e "s:\(_OBJCOPY_PATH\s*=\).*$:\1 $(tc-getOBJCOPY):" \
+			-e "s:\(_RC_PATH\s*=\).*$:\1 $(tc-getOBJCOPY):" \
+			-e "s:\(_SLINK_PATH\s*=\).*$:\1 $(tc-getAR):" \
+			-i "${UDK_WORKSPACE}/Conf/tools_def.txt" \
+			|| die "Failed to prepare tools definition file"
+	fi
+}
+
+src_compile() {
+	# Ensure environment is set
+	if ! use gnuefi; then
+		. udk-workspace "${UDK_WORKSPACE}" || die
+	fi
+
+	# Update fs targets depending on uses
+	local fs fs_names=()
+	for fs in ${FS_USE}; do
+		fs=${fs#+}
+		if use "${fs}"; then
+			fs_names+=("${fs}$(usex gnuefi _gnuefi '')")
+		fi
+	done
+
+	# Prepare flags
+	local make_flags=(
+		ARCH="${BUILDARCH}"
+		CC="$(tc-getCC)"
+		AS="$(tc-getAS)"
+		LD="$(tc-getLD)"
+		AR="$(tc-getAR)"
+		RANLIB="$(tc-getRANLIB)"
+		OBJCOPY="$(tc-getOBJCOPY)"
+		GNUEFILIB="/usr/$(get_libdir)"
+		EFILIB="/usr/$(get_libdir)"
+		EFICRT0="/usr/$(get_libdir)"
+		EDK2BASE="${UDK_WORKSPACE}"
+		EDK2_DRIVER_BASENAMES="${fs_names[@]}"
+		FILESYSTEMS="${fs_names[@]}"
+		FILESYSTEMS_GNUEFI="${fs_names[@]}"
+	)
+	if use custom-cflags; then
+		make_flags=(CFLAGS="${CFLAGS}" "${make_flags[@]}")
+	fi
+
+	emake "${make_flags[@]}" $(usex gnuefi gnuefi edk2)
+	# Filesystems do not support parallel make
+	if use gnuefi; then
+		emake -j1 "${make_flags[@]}" -C "${S}"/filesystems gnuefi
+	else
+		emake -j1 "${make_flags[@]}" fs_edk2
+	fi
+}
+
+src_install() {
+	exeinto "/usr/share/${P}"
+	doexe refind-install
+	dosym "../share/${P}/refind-install" "/usr/sbin/refind-install"
+
+	dodoc "${S}"/{COPYING.txt,LICENSE.txt,CREDITS.txt}
```

Comment:
1. S is the current directory, isn't it? Here and below.
2. Why are you installing license files? That's against Gentoo policy.